src/SpannerDriver.php

```diff
@@ -74,12 +71,18 @@ class SpannerDriver implements Driver
     /** @var SpannerClient */
     private $spannerClient;
 
+    /** @var SessionPoolFactory */
+    private $sessionPoolFactory;
+
     public function __construct(
         SpannerClientFactory $spannerClientFactory = null,
-        SessionPoolInterface $sessionPool = null
-    ) {
+        SessionPoolInterface $sessionPool = null,
+        SessionPoolFactory $sessionPoolFactory = null
+    )
+    {
```

Comment:
[PSR-2](https://www.php-fig.org/psr/psr-2/#44-method-arguments): `When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.`
```suggestion
    ) {
```

Reply:
Done! :)